blog/forms.py

```diff
@@ -0,0 +1,14 @@
+"""Creating and editing forms."""
+from django import forms
+
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Class form based on the model."""
```

Comment:
i see that it's a class, it's stated one line above. anything useful you want to tell me with this?